lib/Sema/CSGen.cpp

```diff
@@ -1836,14 +1839,19 @@ namespace {
       auto locator = CS.getConstraintLocator(expr);
       auto contextualType = CS.getContextualType(expr);
       auto contextualPurpose = CS.getContextualTypePurpose(expr);
-      auto openedType =
-          CS.openOpaqueType(contextualType, contextualPurpose, locator);
 
       // If a contextual type exists for this expression and is a dictionary
       // type, apply it directly.
-      Optional<std::pair<Type, Type>> dictionaryKeyValue;
-      if (openedType && (dictionaryKeyValue =
-                             ConstraintSystem::isDictionaryType(openedType))) {
+      if (contextualType && ConstraintSystem::isDictionaryType(contextualType)) {
+        // Now that we know we're actually going to use the type, get the verion
```

Comment:
```suggestion
        // Now that we know we're actually going to use the type, get the version
```

Reply:
Thanks, @LucianoPAlmeida! I think github just needs to implement a spellchecker for PR requests :)

Reply:
haha, true 😆

Reply:
Alas, this line runs right up against the 80th column here so I’ll have to rebreak the lines 🤦